apps/sailfish/qml/pages/KodiPage.qml

```diff
@@ -122,7 +129,10 @@ Page {
                 }
 
                 var target = kodiMenuModel.get(index).target;
-                if (target === "quit") {
+                if (target === "logOff") {
```

Comment:
Not entirely sure how the sailfish ui looks like, but this reads a bit odd. There is a menu item called "Log off". If the user selects that, it opens the ProfileSelectionDialog which has a title named "Log in".

I think it would be better to to name this "Change user" and then the title of the dialog "Select user"